### PR TITLE
[Doc] PR for Improve Platform uninstall instructions

### PR DESCRIPTION
**Pull request by lizayugabyte**
Partially addresses https://yugabyte.atlassian.net/browse/PLAT-2740

Note that the Replicated rm command (step 3) doesn't work, according to dev. Platform PM is trying to decide what to do about this part of the document. Update: decided to remove the Docker section for now. The doc ticket in Jira will remain open as "in progress".

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md [hunk 1]
```diff
@@ -12,63 +12,72 @@ isTocNested: true
...
-1. To remove the Yugabyte Platform, run the `helm delete` command:
+1. To remove the Yugabyte Platform, execute the following helm command:
 
     ```sh
-    $ helm del yw-test -n yw-test
+    helm del yw-test -n yw-test
     ```
 
-    A message displays that the Yugabyte Platform release and the namespace is deleted.
+    <br>You should see a message similar to the following, notifying you that the Yugabyte Platform release and the namespace is deleted:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md:68]: namespace is deleted? but following step is to delete namespace

**Review by ddhodge — APPROVED** [review]
2 comments

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md]: Fixed

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md [hunk 2]
```diff
@@ -12,63 +12,72 @@ isTocNested: true
...
-6. Uninstall Replicated by following instructions documented in [Removing Replicated](https://help.replicated.com/docs/native/customer-installations/installing-via-script/#removing-replicated).
+6. Uninstall Replicated by following instructions provided in [Removing Replicated](https://help.replicated.com/docs/native/customer-installations/installing-via-script/#removing-replicated).
 
 ## Uninstall in Kubernetes environments
+-->
+
+You can uninstall Yugabyte Platform in Kubernetes, as follows:
 
-1. To remove the Yugabyte Platform, run the `helm delete` command:
+1. To remove the Yugabyte Platform, execute the following helm command:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/uninstall-software.md]: fixed

**Pull request merged**